src/functionalTest/java/uk/gov/hmcts/reform/sendletter/controllers/SendLetterTest.java

```diff
@@ -74,7 +76,11 @@ public void should_return_200_when_single_letter_is_sent() throws Exception {
             .when(letterRepository)
             .save(any(WithServiceNameAndId.class), any(Instant.class), anyString());
 
-        send(readResource("letter.json")).andExpect(status().isOk());
+        MvcResult result = send(readResource("letter.json"))
+            .andExpect(status().isOk())
+            .andReturn();
+
+        assertThat(result.getResponse().getContentAsString()).isNotNull();
```

Comment:
could just append another mockMvc matcher `.andExcpet(content().string(is("uuid regex")))` or if some uuid checker exists :)

Reply:
ok I had a look at this there is no uuid checker exists.We can use `UUID.fromString(letterId);` but that won't be of much use.I agree with @rkondratowicz that controller should just pass what is received from service.